src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -5,7 +5,6 @@ import {useRoute} from '@react-navigation/native';
 
 import AnonymousMessageRepo from '../../service/repo/anonymousMessageRepo';
 import ChannelList from '../../database/schema/ChannelListSchema';
-import ChannelListMemberSchema from '../../database/schema/ChannelListMemberSchema';
 import ChatSchema from '../../database/schema/ChatSchema';
 import SignedMessageRepo from '../../service/repo/signedMessageRepo';
 import UseLocalDatabaseHook from '../../../types/database/localDatabase.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `ChannelListMemberSchema` has been removed. Ensure that this is intentional and that there are no remaining references to `ChannelListMemberSchema` in the file, as this could lead to undefined reference errors.
```diff
- import ChannelListMemberSchema from '../../database/schema/ChannelListMemberSchema';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->